lib/runcible/models/yum_importer.rb

```diff
@@ -3,6 +3,13 @@ module Models
     class YumImporter < Importer
       ID = 'yum_importer'
       REPO_TYPE = 'rpm-repo'
+      DOWNLOAD_POLICIES = [
+        DOWNLOAD_IMMEDIATE = 'immediate',
+        DOWNLOAD_ON_DEMAND = 'on_demand',
+        DOWNLOAD_BACKGROUND = 'background'
```

Comment:
Wouldn't it be YumImporter::DOWNLOAD_POLICIES::DOWNLOAD_IMMEDIATE ? Or is this code chunk just confusing me all together about what it is actually doing?